standard/tournament_structure.lua

```diff
@@ -273,7 +273,7 @@ function TournamentStructure.fetchGroupTableEntries(group)
 			local sortValue = tonumber(record.extradata.slotindex) or tonumber(record.placement)
 			sortValue = placeMapping[sortValue] or sortValue
 			record.extradata.slotindex = sortValue
-			record.placement = sortValue
+			record.placement = tostring(sortValue)
```

Comment:
And this doesn't break anything?

Reply:
works fine in my tests
also any usage of `.placement` is either directly in a tonumber again or passed to a function that does a tonumber anyways